pkg/operator/csidriveroperator/csioperatorclient/manila.go

```diff
@@ -40,17 +40,31 @@ func GetOpenStackManilaOperatorConfig(isHypershift bool, clients *csoclients.Cli
 
 	if !isHypershift {
 		csiDriverConfig.StaticAssets = []string{
-			"csidriveroperators/openstack-manila/standalone/generated/v1_namespace_openshift-cluster-csi-drivers.yaml",
-			"csidriveroperators/openstack-manila/standalone/generated/v1_serviceaccount_manila-csi-driver-operator.yaml",
-			"csidriveroperators/openstack-manila/standalone/generated/rbac.authorization.k8s.io_v1_role_manila-csi-driver-operator-role.yaml",
-			"csidriveroperators/openstack-manila/standalone/generated/rbac.authorization.k8s.io_v1_rolebinding_manila-csi-driver-operator-rolebinding.yaml",
+			"csidriveroperators/openstack-manila/standalone/generated/v1_namespace_openshift-manila-csi-driver.yaml",
+			"csidriveroperators/openstack-manila/standalone/generated/openshift-cluster-csi-drivers_v1_serviceaccount_manila-csi-driver-operator.yaml",
+			"csidriveroperators/openstack-manila/standalone/generated/openshift-cluster-csi-drivers_rbac.authorization.k8s.io_v1_role_manila-csi-driver-operator-role.yaml",
+			"csidriveroperators/openstack-manila/standalone/generated/openshift-cluster-csi-drivers_rbac.authorization.k8s.io_v1_rolebinding_manila-csi-driver-operator-rolebinding.yaml",
 			"csidriveroperators/openstack-manila/standalone/generated/rbac.authorization.k8s.io_v1_clusterrole_manila-csi-driver-operator-clusterrole.yaml",
 			"csidriveroperators/openstack-manila/standalone/generated/rbac.authorization.k8s.io_v1_clusterrolebinding_manila-csi-driver-operator-clusterrolebinding.yaml",
 		}
-		csiDriverConfig.CRAsset = "csidriveroperators/openstack-manila/standalone/generated/operator.openshift.io_v1_clustercsidriver_manila.csi.openstack.org.yaml"
-		csiDriverConfig.DeploymentAsset = "csidriveroperators/openstack-manila/standalone/generated/apps_v1_deployment_manila-csi-driver-operator.yaml"
+		csiDriverConfig.CRAsset = "csidriveroperators/openstack-manila/standalone/generated/default_operator.openshift.io_v1_clustercsidriver_manila.csi.openstack.org.yaml"
+		csiDriverConfig.DeploymentAsset = "csidriveroperators/openstack-manila/standalone/generated/openshift-cluster-csi-drivers_apps_v1_deployment_manila-csi-driver-operator.yaml"
```

Comment:
Whoops, this probably should have gone into https://github.com/openshift/cluster-storage-operator/pull/514 / an earlier commit in this PR.